addons/crm/models/crm_lead.py

```diff
@@ -997,6 +998,12 @@ def action_set_lost(self, **additional_values):
             self.write(dict(additional_values))
         return res
 
+    @api.model
+    def action_set_lost_with_reason(self):
+        action = self.env["ir.actions.act_window"]._for_xml_id('crm.crm_lead_lost_action')
```

Comment:
Mixing double-quotes and single-quotes.

(Ok, I had nothing else to say, I had to find _something_ ^^)